config/rbac/role.yaml

```diff
@@ -99,7 +98,12 @@ rules:
   - get
   - list
   - patch
-
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  - leases
```

Comment:
Just fyi. These rbac's are controlled by `// +kubebuilder:rbac:groups` pragma statements that generate this RBAC. So by adding it manually here it will probably be clobbered by next controller-gen execution; You'd need to add the proper rbac statement here:
https://github.com/keikoproj/addon-manager/blob/e1aac607f5312ca8d2640962a82cd84b97ab67b4/controllers/addon_controller.go#L111

Reply:
Thanks for pointing this out. Currently, we are using controller-gen@v0.2.5, we should plan to bump to later version.

Reply:
One of the points of adding rbac in yaml file is for user who is going to deploy the controller using pure yaml file. They might not run the step "make manifests."

I think we should have lease in both "marker comment" and yaml.

Reply:
4177242ed4f7af0c1e029985a9fe01adc7f3194e added

Reply:
sort this out https://github.com/keikoproj/addon-manager/pull/96/commits/4177242ed4f7af0c1e029985a9fe01adc7f3194e